_pages/events/winter-school.md

```diff
@@ -10,12 +10,25 @@
 
 <div class="jumbotron" markdown="1">
 
-Winter School 2020 will be January 7, 2020 - February 2, 2020.
-
-The two mandatory lectures are Tuesday, January 7 and Thursday, January 9 from 19:00 - 21:00 (7pm - 9pm) in [26-100](https://whereis.mit.edu?q=26-100).
+Winter School 2021 will happen - virtually - from January 7, 2021 to January 28, 2021. **[Sign up here!](https://mitoc-trips.mit.edu/trips/1378/)**
 
 </div>
 
+### Winter school 2021
+
+Due to the current surge of cases locally and in accordance with [MIT's response to COVID-19](http://web.mit.edu/covid19/) and the various local travel orders, we will not be able to run any in-person event.
+
+Having said that, get stoked: we have a plan for everyone to learn or brush up their **winter adventure skills**, get inspired by **speakers** with diverse and inspiring backgrounds, **raffle** cool prizes, and more!
+
+This year, we'll hold 4 virtual lectures, every Thursday of IAP, from 7pm to 9pm. In more details:
```

Comment:
Stating that lectures are held in Eastern time might be obvious, but clarity never hurts? Especially since many folks are dispersed across the country...